code/src/main/java/frc/robot/commands/navcommands/MoveBackwardsCommand.java

```diff
@@ -0,0 +1,29 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands.navcommands;
+
+import com.google.inject.Inject;
+
+import frc.robot.common.ILogger;
+import frc.robot.common.Location;
+
+/**
+ * Moves the robot backwards based on the distance in meters
+ */
+public class MoveBackwardsCommand extends MoveCommand {
+
+    @Inject
```

Comment:
Could you add todo stubs to the commands saying we still need to implement functionality?

Reply:
Actually if the gotoPositionCommand is working then this is done. This isn't a mock command.

Reply:
Only to the extent that we won't have good auton. We can code up an 'alternative' command for the auton, but that shouldn't affect anything else. We can continue to build out other strategies while that command is built.